torch_xla/csrc/ops/unique2.cpp

```diff
@@ -0,0 +1,42 @@
+#include "torch_xla/csrc/ops/unique2.h"
+
+#include "torch_xla/csrc/lowering_context.h"
+#include "torch_xla/csrc/tensor_util.h"
+#include "torch_xla/csrc/xla_lower_util.h"
+
+namespace torch_xla {
+namespace {
+
+xla::Shape NodeOutputShape(const torch::lazy::Value& input) {
+  xla::Shape input_shape = GetXlaShape(input);
+  int64_t num_elements = xla::ShapeUtil::ElementsIn(input_shape);
+  xla::PrimitiveType indices_type = GetShapeDimensionType(/*device=*/nullptr);
```

Comment:
Same question here, do we want the indices type to be S32 or S64? Though returning S32 does not break any tests.

Reply:
I would leave most of the op in S32 to avoid extra cost lol. I think eventually we want to make `setDimensionSize` and `getDimensionSize` handle S64, which I don't think is super complicated, but someone need to work on it...